portfolio/src/main/webapp/index.html

```diff
@@ -4,16 +4,110 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+        <!-- Header Start -->
+        <ul class="nav justify-content-end" id="navBar">
+            <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#intro">Home</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#about">About</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#project">Project</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#contact" >Contact</a>
+            </li>
+            <!-- <li class="nav-item">
```

Comment:
Remove commented out code for neatness :)

Reply:
Resolved!